src/Pages/CategoriePage/ItemCategory.js

```diff
@@ -0,0 +1,89 @@
+import React, { Component } from 'react';
+import './ItemCategory.scss';
+import ShirtList from './ShirtList';
+
+class ItemCategory extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      click: false,
+      imgChange: true,
+      harucheck: true,
+    };
+  }
+
+  upDown = () => {
+    const { click } = this.state;
+    this.setState({ click: !click });
+  };
+
+  render() {
+    const { click } = this.state;
+    const { handleItemFilter, handleSaleFilter } = this.props;
+    const upDownImg = click ? '../../../images/up.png' : '../../../images/down.png';
+    return (
+      <div className="ItemCategory">
+        <nav className="nav">
+          <div className="navBox">
+            <div className="mainCategory">
+              <div className="categoryTitle">상품옵션</div>
+              <div className="categoryList">
+                <input type="checkbox" onChange={handleItemFilter} />
+                <img src="./images/haru2.png" alt="하루배송 사진" />
+              </div>
+              <div className="categoryList">
+                <input type="checkbox" onChange={handleSaleFilter} />
+                <span>세일</span>
+              </div>
+            </div>
+          </div>
+          <div className="categoryBox">
+            <div className="category">CATEGORY</div>
+            <div className="categoryList">전체</div>
+            <div className="categoryOuterBox">
+              <div className="categoryListSub">아우터</div>
+              <div>
+                <img className="arrow" src="./images/down.png" alt="down" />
+              </div>
+            </div>
+            <div className="categoryOuterBox" onClick={() => this.upDown()}>
```

Comment:
```suggestion
            <div className="categoryOuterBox" onClick={() => this.setState({ click: !click })}>
```

한 번만 호출되는 함수라면 함수를 따로 정의하기 보다는 바로 로직을 실행시켜주는 방법도 있습니다

Reply:
오오 꿀팁 감사합니다!